src/constants/routes.ts

```diff
@@ -69,18 +69,18 @@ export const routes = {
   myPatches: `${paths.user}/${PageNames.Patches}`,
   patch: `${paths.patch}/:id`,
   preferences: paths.preferences,
-  projectPatches: `${paths.project}/:id/${PageNames.Patches}`,
-  projectSettings: `${paths.project}/:identifier/${PageNames.Settings}`,
+  projectPatches: `${paths.project}/:projectIdentifier/${PageNames.Patches}`,
+  projectSettings: `${paths.project}/:projectIdentifier/${PageNames.Settings}`,
```

Comment:
This is a somewhat annoying route because the identifier here could be for a project _or_ repo (the previously used `identifier` could signify either). But I think there's benefit to keeping the path segment consistent in our routes so the change makes sense.

Reply:
Ohh that is a good point, I did not consider this 😔 I think it's okay for now for the same reason we call it "Project Settings", yet it also technically contains repo settings

we can change it later too! :]